Sources/Commands/SwiftPackageTool.swift

```diff
@@ -190,22 +190,25 @@ extension SwiftPackageTool {
         }
     }
 
-    struct Init: SwiftCommand {
-        static let configuration = CommandConfiguration(
+    public struct Init: SwiftCommand {
```

Comment:
why does this need to be public?

Reply:
If we want to use `AssertHelp` we need access to the commands.

Reply:
you can use `@testable import` to use test internal structures and functions. that is better than exposing more surface area in the public API.